ztp/gitops-subscriptions/argocd/deployment/argocd-openshift-gitops-patch.json

```diff
@@ -17,8 +17,7 @@
       "volumes": [
         {
           "name": "kustomize",
-          "readOnly": false,
-          "path": "/.config"
+          "emptyDir": {}
```

Comment:
What does this additional field do? (is it needed?)

Reply:
Yep looks like it's in use below e.g 
```
...
"volumeMounts": [
        {
          "name": "kustomize",
          "mountPath": "/.config"
        }
      ]
...
```